packages/components/src/progress-bar/index.tsx

```diff
@@ -46,6 +46,53 @@ function UnforwardedProgressBar(
 	);
 }
 
+/**
+ * A simple horizontal progress bar component.
```

Comment:
I've always felt we duplicate so much README and add overhead for maintaining these docs in the future 😕 I wonder if we can improve/automate that somehow - perhaps a command to allow the README to inherit content from here?

Reply:
My initial plans were to abandon the manual readmes altogether once the Storybook was "complete" with all the type data and documentation. I do consider it pretty much complete now. But @DaniGuardiola mentioned some kind of TSDoc to markdown docgen he has up his sleeve, so that could be another option.

Reply:
ref: https://github.com/Create-Inc/archetype/?tab=readme-ov-file#createincarchetype-docgen

Reply:
I see. It would be fun to try something like this out here @DaniGuardiola 👍

Reply:
Yeah, I share the non-DRY feelings here, too. A way to extract and keep everything from/in a central place would be nice 👍🏻